agent/timings.js

```diff
@@ -90,17 +90,28 @@ function recordLcp() {
 
 function updateLatestLcp(lcpEntry) {
   if (lcp) {
-    var previous = lcp[0]
+    var previous = lcp
```

Comment:
I think this logic with LCP should be kept as before. This is because we want to report the CLS value that was at the time the LCP occurred. LCP is not reported immediately - it may or may not change by the time it is collected.

Not great that our tests did not catch this.

Reply:
this can be changed because it now references CLS directly instead of a passthrough which was out of sync previously (and broke tests).  See [line 75](https://github.com/newrelic/newrelic-browser-agent/pull/108/files#diff-895b7b82ab7cdcd68bb5afe6ba6ad5c1ba82e3d5a628afc88fe078f53ba85d45R75), it used to reference the CLS value as a clone of the value which was passed through like `[lcpEntry, cls]`, and subsequently became out of sync.  Now it directly references the global `var cls` which will always be in sync.